src/index.js

```diff
@@ -247,6 +252,8 @@ const createListItem = (location, status, lng, lat) => {
     openingSoonForReceiving = `<p class="opening-soon"><span data-translation-id="opening_soon">Opening soon!</span> ${openTimeReceiving.format("LT")} <span data-translation-id="for_receiving">for receiving</span></p>`
   }
 
+ const hiddenSearch = hiddenSearchFields.map(field => `<p class="${field}" >${location[field] || ''}</p>`).join('')
```

Comment:
I believe these hiddenSearch fields are currently visible in the cards. The cards seem much larger on staging right now. Was this intended?

Reply:
Staging:
<img width="603" alt="Twin_Cities_Aid_Distribution_Locations" src="https://user-images.githubusercontent.com/2358067/84114360-d8ef5f00-a9f1-11ea-8a6d-9b7646b36c77.png">

Prod:
<img width="627" alt="Twin_Cities_Aid_Distribution_Locations" src="https://user-images.githubusercontent.com/2358067/84114450-fd4b3b80-a9f1-11ea-8fe0-40a192b2fef5.png">